src/analyzer.ts

```diff
@@ -335,7 +335,10 @@ export class Analyzer {
     const scriptCommand = syntaxInfo.scriptNode;
     const wireitConfig = syntaxInfo.wireitConfigNode;
 
-    if (wireitConfig !== undefined && scriptCommand.value !== 'wireit') {
+    if (
+      wireitConfig !== undefined &&
+      ![`wireit`, `yarn run -T wireit`].includes(scriptCommand.value)
```

Comment:
Would prefer `scriptCommand.value !== 'wireit' && scriptCommand.value !== 'yarn run -T -B wireit'`, to avoid the extra array allocation.

Reply:
It's your project, so I'm happy to make whatever changes you prefer.

Does the array allocation matter?  Seems like this runs once at the beginning, where the impact of an array is negligible compared to the benefits to {read/maintain}ability.

Reply:
(with prettier, it ends up like this:
```ts
    if (
      wireitConfig !== undefined &&
      scriptCommand.value !== 'wireit' &&
      scriptCommand.value !== 'yarn run -T -B wireit'
    ) {
```
so the array isn't significantly more readable)